documentation/final-deliverables/step-by-step-guide.md

```diff
@@ -62,15 +62,25 @@ Install Node:
 
 ## Maintenance Issues
 
-Gmail account details: (to be added)
+### Gmail Account (Nodemailer) Details
```

Comment:
I was looking into Nodemailer, and [this](https://nodemailer.com/usage/using-gmail/#:~:text=Gmail%20has%20a%20limit%20of,recipients%20a%20day%20for%20free.) might be good to note as a limitation (assuming that in the future the platform grows and needs to send >500 emails per day)

Reply:
Sounds good! will write that!